contracts/RealmDiamond/facets/AlchemicaFacet.sol

```diff
@@ -232,7 +233,7 @@ contract AlchemicaFacet is Modifiers {
 
     //gotchi CANNOT have active listing for lending
     require(
-      !diamond.isAavegotchiListed(uint32(_gotchiId)) || diamond.isAavegotchiLent(uint32(_gotchiId)),
+      !diamond.isAavegotchiListed(uint32(_gotchiId)),
       "AavegotchiDiamond: Gotchi CANNOT have active listing for lending"
     );
```

Comment:
I would remove this too, as the Aavegotchi is not transferred when listed, nor we can predict the roles of the listing